.github/workflows/build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Node Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
action: remove these lines. we should only build on the master branch (i.e. merged from a PR) but not on a PR created